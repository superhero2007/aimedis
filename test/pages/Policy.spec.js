import React from 'react';
import { shallow } from 'enzyme';

import Policy from 'pages/Policy';

function setup() {
  return shallow(<Policy />);
}

describe('Policy', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
