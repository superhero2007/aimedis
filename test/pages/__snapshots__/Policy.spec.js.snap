// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Policy should render properly 1`] = `"<div class=\\"app__policy app__route\\"><div class=\\"container\\"><section class=\\"app__policy__section1\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-7\\"><div class=\\"app__policy__section1__content\\"><p class=\\"app__policy__section1__content__title\\">Policy</p><p class=\\"app__policy__section1__content__description\\">Among the types of Personal Data that this Application collects, by itself or through third parties, there are: general activity data, body measurements &amp; indexes, movement activity, food related activity , sleeping activity, heart rate and other vital data, blood type, gender, date of birth, Cookies, Usage Data, first name, last name, phone number, address, username, email address, password, company name, profession, fax number, country, state, province, ZIP/Postal code, city, picture, workplace, various types of Data, field of activity, User ID, marital status, website, billing address, house number, prefix , language, profile picture, budget and academic background.</p><p class=\\"app__policy__section1__content__description\\">Complete details on each type of Personal Data collected are provided in the dedicated sections of this privacy policy or by specific explanation texts displayed prior to the Data collection.<br/>The Personal Data may be freely provided by the User, or, in case of Usage Data, collected automatically when using this Application.</p><p class=\\"app__policy__section1__content__subtitle1\\">Mode and place of processing the Date</p><p class=\\"app__policy__section1__content__subtitle2\\">Methods of processing</p><p class=\\"app__policy__section1__content__description\\">Any use of Cookies – or of other tracking tools – by this Application or by the owners of third party services used by this Application serves the purpose of providing the service required by the User, in addition to any other purposes described in the present document and in the Cookie Policy, if available.<br/>Users are responsible for any third party Personal Data obtained, published or shared through this Application and confirm that they have the third party&#x27;s consent to provide the Data to the Owner.</p></div></div></div></section><section class=\\"app__policy__mask d-none d-lg-block\\"><div class=\\"app__policy__mask__bottom\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></section></div></div>"`;
